core/arch/arm/kernel/boot.c

```diff
@@ -188,6 +188,29 @@ static void init_vfp_nsec(void)
 }
 #endif
 
+/*
+ * Check for supported Crypto Extensions (ARMv8 aarch32/aarch64)
+ * In case one of instructions is not supported false is returned.
+ */
+static bool check_cpuid_ce(void)
+{
+	uint32_t isar5 = read_isar5();
+
+	if (!(isar5 | ID_ISAR5_AES))
```

Comment:
maybe also makes sense to additionally check for algo specific flag like `CFG_CRYPTO_AES_ARM_CE` in each if statement, however I'm not sure if it's the case when one algo from crypto extensions is not available, but others are.